### PR TITLE
Add alt text (and aria-hidden) for seattle logos.

### LINKED ISSUE #1473: Give logo image an `alt` attribute
The Seattle logo image, found in the header bar, lacks an `alt` attribute, which is important for accessibility. Let's give it one.

image

## Original tester's report

> ACTUAL RESULT:
Voice Over: Screen reader is announcing 'Seattle' image logo as 'Seattle-logo_horizontal_blue-white_small.png, image'
ChromeVox: Screen reader is announcing 'Seattle' image logo as 'To get missimg image description, open the context menu. Image'
JAWS/Chrome: Screen reader is announcing 'Seattle' image logo as 'To get missimg image description, open the context menu. Unlabeled graphic'
> EXPECTED RESULT: If it is a decorative image, screen reader focus should not be set on the 'Seattle' logo image. If the image is conveying valid information, provide appropriate description for the 'Seattle' image.

### PR DESCRIPTION
**Pull request by ndmckinley**
### Description
These don't need to be read by screen readers.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
Fixes #1473.

### CODE REVIEW

File: universal-application-tool-0.0.1/app/views/LoginForm.java [hunk 1]
```diff
@@ -61,6 +61,8 @@ private ContainerTag mainContent(Messages messages) {
         this.layout
             .viewUtils
             .makeLocalImageTag("ChiefSeattle_Blue")
+            .withAlt("City of Seattle Logo")
```
Comment (kendrickt) [inline universal-application-tool-0.0.1/app/views/LoginForm.java:64]: (blocking) these should be i18nized right?

Comment (ndmckinley) [reply universal-application-tool-0.0.1/app/views/LoginForm.java:64]: Not worth it imo - they are not going to be read aloud, due to `aria-hidden: true`.

**Review by hmhsu — APPROVED** [review]

**Comment by hmhsu** [comment]
I think the logo conveys the message that the website is legit and official. I think it would be nice to somehow convey that message, not necessarily through reading aloud the logo alt text though.

**Comment by ndmckinley** [comment]
I'll wait for @kendrickt to chime in since he made a blocking comment.

**Review by kendrickt — APPROVED** [review]

**Pull request merged**